avocado/utils/asset.py

```diff
@@ -285,20 +284,17 @@ def _verify(self, asset_path):
         value as the hash of the asset_file, otherwise return False.
         :rtype: bool
         """
-        if not self.asset_hash:
-            return True
-        if self._get_hash_from_file(asset_path) == self.asset_hash:
-            return True
-        else:
-            return False
+        return bool(not self.asset_hash or
+                    (self.asset_hash and
+                     self._get_hash_from_file(asset_path) == self.asset_hash))
```

Comment:
Why is this change needed? I mean, this is way less readable and the second `self.asset_hash` check inside the `boll()` is redundant. I'd just remove the else (while keeping the final return), but if you insist in changing this, you should consider using [any()](https://docs.python.org/3.6/library/functions.html#any).

Reply:
Okay, I'll just remove the `else` on v2.